src/stores/Frame/RegionSetStore.ts

```diff
@@ -12,11 +12,19 @@ export enum RegionMode {
     CREATING
 }
 
+export enum RegionsOpacity {
+    Visible = 1,
+    Transparent = 0.5,
```

Comment:
I think it would make sense to call this `SemiTransparent`. Transparent implies fully transparent.